cmake/Dependencies.cmake

```diff
@@ -411,6 +411,8 @@ if(USE_CUDA)
     endif()
     if(CAFFE2_USE_CUDNN)
       list(APPEND Caffe2_PUBLIC_CUDA_DEPENDENCY_LIBS caffe2::cudnn)
+    else()
+      caffe2_update_option(USE_CUDNN OFF)
```

Comment:
Quick question: what about CAFFE2_USE_CUDNN?

Reply:
CAFFE2_USE_CUDNN is updated in cmake/public/cuda.cmake, so we should be good.